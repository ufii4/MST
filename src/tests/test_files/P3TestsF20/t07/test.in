n 256
i 0;1;0.10
i 1;2;0.11
i 2;3;0.12
i 3;4;0.13
i 4;5;0.14
i 5;6;0.15
i 6;7;0.16
i 7;8;0.17
i 8;9;0.18
i 9;10;0.19
i 10;11;0.20
i 11;12;0.21
i 12;13;0.22
i 13;14;0.23
i 14;15;0.24
i 15;16;0.25
i 16;17;0.26
i 17;18;0.27
i 18;19;0.28
i 19;20;0.29
i 20;21;0.30
i 21;22;0.31
i 22;23;0.32
i 23;24;0.33
i 24;25;0.34
i 25;26;0.35
i 26;27;0.36
i 27;28;0.37
i 28;29;0.38
i 29;30;0.39
i 30;31;0.40
i 31;32;0.41
i 32;33;0.42
i 33;34;0.43
i 34;35;0.44
i 35;36;0.45
i 36;37;0.46
i 37;38;0.47
i 38;39;0.48
i 39;40;0.49
i 40;41;0.50
i 41;42;0.51
i 42;43;0.52
i 43;44;0.53
i 44;45;0.54
i 45;46;0.55
i 46;47;0.56
i 47;48;0.57
i 48;49;0.58
i 49;50;0.59
i 50;51;0.60
i 51;52;0.61
i 52;53;0.62
i 53;54;0.63
i 54;55;0.64
i 55;56;0.65
i 56;57;0.66
i 57;58;0.67
i 58;59;0.68
i 59;60;0.69
i 60;61;0.70
i 61;62;0.71
i 62;63;0.72
i 63;64;0.73
i 64;65;0.74
i 65;66;0.75
i 66;67;0.76
i 67;68;0.77
i 68;69;0.78
i 69;70;0.79
i 70;71;0.80
i 71;72;0.81
i 72;73;0.82
i 73;74;0.83
i 74;75;0.84
i 75;76;0.85
i 76;77;0.86
i 77;78;0.87
i 78;79;0.88
i 79;80;0.89
i 80;81;0.90
i 81;82;0.91
i 82;83;0.92
i 83;84;0.93
i 84;85;0.94
i 85;86;0.95
i 86;87;0.96
i 87;88;0.97
i 88;89;0.98
i 89;90;0.99
i 90;91;0.10
i 91;92;0.11
i 92;93;0.12
i 93;94;0.13
i 94;95;0.14
i 95;96;0.15
i 96;97;0.16
i 97;98;0.17
i 98;99;0.18
i 99;100;0.19
i 100;101;0.20
i 101;102;0.21
i 102;103;0.22
i 103;104;0.23
i 104;105;0.24
i 105;106;0.25
i 106;107;0.26
i 107;108;0.27
i 108;109;0.28
i 109;110;0.29
i 110;111;0.30
i 111;112;0.31
i 112;113;0.32
i 113;114;0.33
i 114;115;0.34
i 115;116;0.35
i 116;117;0.36
i 117;118;0.37
i 118;119;0.38
i 119;120;0.39
i 120;121;0.40
i 121;122;0.41
i 122;123;0.42
i 123;124;0.43
i 124;125;0.44
i 125;126;0.45
i 126;127;0.46
i 127;128;0.47
i 128;129;0.48
i 129;130;0.49
i 130;131;0.50
i 131;132;0.51
i 132;133;0.52
i 133;134;0.53
i 134;135;0.54
i 135;136;0.55
i 136;137;0.56
i 137;138;0.57
i 138;139;0.58
i 139;140;0.59
i 140;141;0.60
i 141;142;0.61
i 142;143;0.62
i 143;144;0.63
i 144;145;0.64
i 145;146;0.65
i 146;147;0.66
i 147;148;0.67
i 148;149;0.68
i 149;150;0.69
i 150;151;0.70
i 151;152;0.71
i 152;153;0.72
i 153;154;0.73
i 154;155;0.74
i 155;156;0.75
i 156;157;0.76
i 157;158;0.77
i 158;159;0.78
i 159;160;0.79
i 160;161;0.80
i 161;162;0.81
i 162;163;0.82
i 163;164;0.83
i 164;165;0.84
i 165;166;0.85
i 166;167;0.86
i 167;168;0.87
i 168;169;0.88
i 169;170;0.89
i 170;171;0.90
i 171;172;0.91
i 172;173;0.92
i 173;174;0.93
i 174;175;0.94
i 175;176;0.95
i 176;177;0.96
i 177;178;0.97
i 178;179;0.98
i 179;180;0.99
i 180;181;0.10
i 181;182;0.11
i 182;183;0.12
i 183;184;0.13
i 184;185;0.14
i 185;186;0.15
i 186;187;0.16
i 187;188;0.17
i 188;189;0.18
i 189;190;0.19
i 190;191;0.20
i 191;192;0.21
i 192;193;0.22
i 193;194;0.23
i 194;195;0.24
i 195;196;0.25
i 196;197;0.26
i 197;198;0.27
i 198;199;0.28
i 199;200;0.29
i 200;201;0.30
i 201;202;0.31
i 202;203;0.32
i 203;204;0.33
i 204;205;0.34
i 205;206;0.35
i 206;207;0.36
i 207;208;0.37
i 208;209;0.38
i 209;210;0.39
i 210;211;0.40
i 211;212;0.41
i 212;213;0.42
i 213;214;0.43
i 214;215;0.44
i 215;216;0.45
i 216;217;0.46
i 217;218;0.47
i 218;219;0.48
i 219;220;0.49
i 220;221;0.50
i 221;222;0.51
i 222;223;0.52
i 223;224;0.53
i 224;225;0.54
i 225;226;0.55
i 226;227;0.56
i 227;228;0.57
i 228;229;0.58
i 229;230;0.59
i 230;231;0.60
i 231;232;0.61
i 232;233;0.62
i 233;234;0.63
i 234;235;0.64
i 235;236;0.65
i 236;237;0.66
i 237;238;0.67
i 238;239;0.68
i 239;240;0.69
i 240;241;0.70
i 241;242;0.71
i 242;243;0.72
i 243;244;0.73
i 244;245;0.74
i 245;246;0.75
i 246;247;0.76
i 247;248;0.77
i 248;249;0.78
i 249;250;0.79
i 250;251;0.80
i 251;252;0.81
i 252;253;0.82
i 253;254;0.83
i 254;255;0.84
i 255;0;0.85
edge_count
mst
clear
edge_count
mst
exit
