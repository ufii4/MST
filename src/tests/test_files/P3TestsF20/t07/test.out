success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
edge count is 256
mst 133.21
success
edge count is 0
not connected
